### ISSUE TITLE
shortcodes are not compatible in dashboard widget

### ISSUE THREAD

**Issue by sridhargoshika** [AyeCode/invoicing#533]
Payment forms shortcode or [wpinv-history] are not working in admin/main dashboard page.

Here is the code in functions.php:
`add_action('wp_dashboard_setup', 'my_dashboard_widgets');

function my_dashboard_widgets() {
global $wp_meta_boxes;
wp_add_dashboard_widget('custom_help_widget', 'Product Request', 'my_product_request_dashboard_widget');
}

function my_product_request_dashboard_widget()
{	
echo do_shortcode('[getpaid form=83]')	;
} 

function my_invoice_dashboard_widget() {
echo do_shortcode('[wpinv_history]');
}`

**Comment by picocodes** [comment]
@sridhargoshika your code works for me. Do you see any errors when you add it to your functions.php file? Most importantly, does the dashboard widget appear?

**Comment by sridhargoshika** [comment]
> @sridhargoshika your code works for me. Do you see any errors when you add it to your functions.php file? Most importantly, does the dashboard widget appear?

Here is my output, Card details windows is also disappeared and PayNow not working.
![image](https://user-images.githubusercontent.com/33679589/110092329-7e880100-7dbf-11eb-8dd3-6da416f858c4.png)

**Comment by picocodes** [comment]
Yeah, I see. Our payment scripts are set to only load on the frontend. Try the following code instead (which forces our scripts to load on the backend too).

```php
add_action('wp_dashboard_setup', 'my_dashboard_widgets');

function my_dashboard_widgets() {
wp_add_dashboard_widget('custom_help_widget', 'Product Request', 'my_product_request_dashboard_widget');
}

function my_product_request_dashboard_widget()
{
echo do_shortcode('[getpaid form=884]') ;
}

function my_invoice_dashboard_widget() {
echo do_shortcode('[wpinv_history]');
}

add_action( 'admin_enqueue_scripts', 'my_admin_load_scripts' );
function my_admin_load_scripts() {
    $screen = get_current_screen();

    if ( ! empty( $screen->id ) && 'dashboard' == $screen->id ) {
        getpaid()->enqueue_scripts();
    }

}

add_filter( 'aui_screen_ids','my_admin_add_aui_screens' );
function my_admin_add_aui_screens( $screens ) {
    $screens[] = 'dashboard';
    return $screens;
}

```

**Comment by sridhargoshika** [comment]
I tried your code. It seems PayNow button working correctly. But invoicing-Pagination not working. It showing first page when I click on 2 page.

![image](https://user-images.githubusercontent.com/33679589/110112802-34f7e000-7dd8-11eb-864f-f373c1f9853f.png)

**Comment by sridhargoshika** [comment]
Any update on this issue
